cmake/find/Findodb.cmake

```diff
@@ -0,0 +1,108 @@
+# ----------------------------------------------------------------------
```

Comment:
> Maybe it was something to do with RPATH, I have to read it up to understand better

Probably yes. See http://www.cmake.org/Wiki/CMake_RPATH_handling. The good news is that we can fix it without modifying native build instructions. We just need to add extra step with `install_name_tool` command which will fix RPATH of library/executable so package can be relocatable.